sql_generators/glean_usage/events_stream.py

```diff
@@ -0,0 +1,62 @@
+"""Generate events stream queries for Glean apps."""
+
+import re
+
+from sql_generators.glean_usage.common import GleanTable
+
+TARGET_TABLE_ID = "events_stream_v1"
+PREFIX = "events_stream"
+# Only generate it for Fenix for now
+APP_ID_ALLOW = {
+    "org_mozilla_firefox",
+    "org_mozilla_fenix",
+    "org_mozilla_fenix_beta",
+}
+DATASET_ALLOW = {
+    "fenix",
+}
```

Comment:
It would be preferable to have this kind of configuration live in here: https://github.com/mozilla/bigquery-etl/blob/be60f5aa56a3ae53123149a4812205a32ebc9462/bqetl_project.yaml#L431

Reply:
This PR looks good, just moving the configuration here in the config file is the biggest todo to make this move ahead